Source/ablastr/fields/PoissonSolver.H

```diff
@@ -226,12 +225,14 @@ computePhi (amrex::Vector<amrex::MultiFab*> const & rho,
         // one of the axes of the grid, i.e. that only *one* of the
         // components of `beta` is non-negligible. // we use this
 #if defined(WARPX_DIM_RZ)
-        linop.setSigma({0._rt, 1._rt-beta_solver[1]*beta_solver[1]});
+        linop.setSigma({
+            -PhysConst::ep0,
```

Comment:
If you're intending to multiply through by `ep0`, this looks like a mistake - the first entry would still be 0, wouldn't it?